src/parser/jobs/smn/modules/Swiftcast.js

```diff
@@ -8,5 +9,45 @@ const MISSED_SEVERITIES = {
 }
 
 export default class Swiftcast extends SwiftcastModule {
+	static handle = 'swiftcast'
+
 	severityTiers = MISSED_SEVERITIES
+	_dwtActive = false
+	_dwtActiveTimestamp = 0
+
+	constructor(...args) {
+		super(...args)
+
+		this.addEventHook('cast', {to: 'player'}, this._onCast)
```

Comment:
The DWT module does have an activeAt method that looks like it would get you the information you are tracking here.  Likewise, you should be able to check getCurrentPet in the pet module for if Phoenix is active to determine if Firebird Trance is running (since we only support level 80).

If that method doesn't turn out to work, I would suggest checking for Deathflare casts rather than damage.  DWT hooks damage for reasons that don't apply to this module.  In the DWT module, it is checking to see if multiple enemies were hit, which was more relevant when trance gave a damage buff.